# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsSelection.cpp
# Commit: 8bf69c3fd814
# Full Hash: 8bf69c3fd81419337cb93f96020bb5e88a333d32
# Author: Jan Varga <Jan.Varga@gmail.com>
# Date: 2012-08-07 13:39:24
# Regressor Bug: 779684
# File Overlap Count: 1
# Description:
#   Bug 779684 - Move FrameIteratorFlags to nsFrameIterator class. r=roc
# ==============================================================================

diff -r 78aaaf0d8185 -r 8bf69c3fd814 layout/generic/nsSelection.cpp
--- a/layout/generic/nsSelection.cpp	Mon Aug 06 14:51:33 2012 -0600
+++ b/layout/generic/nsSelection.cpp	Mon Aug 06 20:27:04 2012 -0400
@@ -1228,7 +1228,7 @@
   nsIFrame *foundFrame = aFrameIn;
 
   nsFrameIterator frameTraversal(mShell->GetPresContext(), aFrameIn,
-                                 eLeaf, FrameIteratorFlags::FLAG_NONE);
+                                 eLeaf, nsFrameIterator::FLAG_NONE);
 
   do {
     *aFrameOut = foundFrame;